test/ref/continuous_test.lst

```diff
@@ -66,6 +66,8 @@ GeneralizedExtremeValue(1.0, 1.0, -1.0)
 GeneralizedExtremeValue(-1.0, 0.5, 0.6)
 
 GeneralizedPareto()
+GeneralizedPareto(1.0)
+GeneralizedPareto(2.0)
```

Comment:
AFAICT currently it is not tested. I guess you have to add it to continuous_test.ref.json as well.

Reply:
> AFAICT currently it is not tested. I guess you have to add it to continuous_test.ref.json as well.

It's already in there, so if it's not being tested I'd be very surprised.

Reply:
The tests (https://github.com/JuliaStats/Distributions.jl/actions/runs/3814509537/jobs/6488867611#step:6:499 and subsequent lines) clearly show that it's not tested. I'm pretty sure it will work if you add a section for `GeneralizedPareto(2.0)` to continuous_test.ref.json.

Reply:
I added reference values for `GeneralizedPareto(2.0)` in https://github.com/JuliaStats/Distributions.jl/pull/1466/commits/6260cebb111d1652b176d41525fb702e9c2fd7bd, let's see if that fixes the tests.

Reply:
It fixed the problem: https://github.com/JuliaStats/Distributions.jl/actions/runs/3878010552/jobs/6613701355#step:6:507